recipe/build.sh

```diff
@@ -15,10 +15,13 @@ else
 fi
 echo "PLATFORM_OPTS $PLATFORM_OPTS"
 
-cmake ${CMAKE_ARGS} -G Ninja -D PORT=Qt -D CMAKE_BUILD_TYPE=Release \
+cmake ${CMAKE_ARGS} \
+    -G Ninja \
+    -D PORT=Qt \
     -D USE_LIBHYPHEN=OFF \
     -D CMAKE_INSTALL_PREFIX="${PREFIX}" \
     -D CMAKE_PREFIX_PATH="${PREFIX}" \
+    -D CMAKE_CXX_STANDARD=17 \
```

Comment:
I don't really get why i need this line, but you might need to add it to windows too